cmd/zetaclientd/start.go

```diff
@@ -233,8 +229,14 @@ func start(_ *cobra.Command, _ []string) error {
 		startLogger.Err(err).Msg("CreateSignerMap")
 		return err
 	}
-	for _, v := range chainClientMap {
-		v.Start()
+
+	if !isNodeActive {
```

Comment:
To clarify, zetaclient will keep going without being impacted by a tombstone in zetacore?

Reply:
yes , since the TSS keyshare is still present it should continue signing outbound ,but the votes will fail 
our ballot threshold is 66% so it should not impact the network